pkg/util/workqueue/timed_queue.go

```diff
@@ -39,24 +39,28 @@ func NewTimedWorkQueue(clock clock.Clock) *TimedWorkQueue {
 // Add adds the obj along with the current timestamp to the queue.
 func (q TimedWorkQueue) Add(obj interface{}) {
 	start := q.clock.Now()
```

Comment:
No, I'm saying if they accidentally call Add instead of AddWithTimestamp, you'll double-wrap, which will give them a nasty surprise.